proto/storage/compliance_operator_v2.proto

```diff
@@ -41,3 +49,31 @@ message ComplianceOperatorRuleV2 {
     string              rationale    = 10;
     string              fixes        = 11;
 }
+
+// Next Tag: 16
+message ComplianceOperatorScanSettingV2 {
+    string                        name                     = 1 [(gogoproto.moretags) = 'sql:"pk"'];
+    bool                          auto_apply_remediations  = 2;
+    bool                          auto_update_remediations = 3;
+    bool                          one_time_scan            = 4;
+    map<string, string>           labels                   = 5;
+    map<string, string>           annotations              = 6;
+    message Profile {
+        string                    profile_id               = 1 [(gogoproto.moretags) = 'search:"-" sql:"fk(ComplianceOperatorProfileV2:id),no-fk-constraint"'];
+    }
+    repeated Profile              profiles                 = 7;
+    message Cluster {
+        string                    cluster_id               = 1 [(gogoproto.moretags) = 'search:"Cluster ID,hidden,store" sql:"fk(Cluster:id),no-fk-constraint,type(uuid)"'];
+        string                    namespace                = 2 [(gogoproto.moretags) = 'search:"Namespace,store"'];
+        string                    namespace_id             = 3 [(gogoproto.moretags) = 'search:"Namespace ID" sql:"fk(NamespaceMetadata:id),no-fk-constraint,type(uuid)"'];
+    }
```

Comment:
Namespace is not a property of cluster. This structure seems a bit absurd. Could you elaborate on structure?

Reply:
yeah left over artifact from a cluster UT that started failing because I didn't have the search category set in scan settings `gen.go`.  Will remove the namespace stuff.